os/drivers/lcd/lcd_dev.c

```diff
@@ -423,13 +424,15 @@ int lcddev_register(struct lcd_dev_s *dev)
 {
 	char devname[16] = { 0, };
 	int ret;
+	char *task_info[2];
```

Comment:
How about `flushing_thread_args`?

Reply:
@abhinav-s235 Could you check my comment above?

Reply:
> How about `flushing_thread_args`?

It has been changed